etc/behat/services/contexts.xml

```diff
@@ -31,8 +31,10 @@
         <parameter key="sylius.behat.context.ui.user.class">Sylius\Behat\Context\Ui\UserContext</parameter>
         <parameter key="sylius.behat.context.ui.channel.class">Sylius\Behat\Context\Ui\ChannelContext</parameter>
         <parameter key="sylius.behat.context.ui.product.class">Sylius\Behat\Context\Ui\ProductContext</parameter>
+        <parameter key="sylius.behat.context.ui.paypal.class">Sylius\Behat\Context\Ui\PaypalContext</parameter>
     </parameters>
     <services>
+        <!--setup-->
```

Comment:
Do we really need those? If we see a need to comment file like this, perhaps we should split the file into: ``contexts/ui.xml``, etc.